proto/internal.proto

```diff
@@ -60,4 +68,25 @@ message GetPlacePageDataResponse {
   repeated string parent_places = 5;
   repeated string similar_places = 6;
   repeated string nearby_places = 7;
+}
+
+
+// Request to get all data in protein page.
+message GetProteinPageDataRequest {
+  // The dcid of the protein
+  string protein = 1;
+}
+// Response to get all data in protein page.
+message GetProteinPageDataResponse {
+  // Message to hold a cohort of nodes that have the same predicate and
+  // direction.
+  message LinkedNodes {
+    string property = 1;
+    PropertyDirection direction = 2;
+    repeated GetProteinPageDataResponse nodes = 3;
+  }
+  // A group of neighbour nodes by predicate.
+  repeated LinkedNodes neighbours = 1;
+  // Value of the node, could be dcid or node string value.
+  string value = 2;
 }
```

Comment:
This seems like a general proto, not protein specific. Can we reuse for others?

Reply:
So, generally, there is a (type, dcid) and response is a graph?

Reply:
Moved the proto to common.proto. 

Right, for disease, there will be a separate config and a subgraph tailored for the disease page.

Reply:
Make the API to be more generic with naming changed from protein to bio